layouts/partials/postlist.html

```diff
@@ -0,0 +1,6 @@
+<div style="font-size: 15px;">
+    Year:
+    {{ .Page.Params.year }}
+    Edition:
+    {{ .Page.Params.edition }}
```

Comment:
That will create:
> Year: 3 Edition 32

While on single post page we have:

> Year 3 - Edition 32

Are we OK with this difference? Do we explicitly want it?
Personally, I would like to re-use this partial in single post, so I want to have same text in both places.